Microsoft.Toolkit.Uwp.SampleApp/SamplePages/FlexPanel/Stepper.xaml.cs

```diff
@@ -0,0 +1,219 @@
+        {
+            if (e.NewValue is double newValue)
+                Value = Math.Max(newValue, Value);
+        }
+        public double MinValue
+        {
+            get => (double)GetValue(MinValueProperty);
+            set => SetValue(MinValueProperty, value);
+        }
+        #endregion MinValue Property
+
+
+        #region MaxValue Property
+        public static readonly DependencyProperty MaxValueProperty = DependencyProperty.Register(
+            nameof(MaxValue),
+            typeof(double),
+            typeof(Stepper),
+            new PropertyMetadata(100.0, new PropertyChangedCallback((d,e) => ((Stepper)d).OnMaxValueChanged(e)))
+        );
+        protected virtual void OnMaxValueChanged(DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is double newValue)
+                Value = Math.Min(newValue, Value);
+        }
+        public double MaxValue
+        {
+            get => (double)GetValue(MaxValueProperty);
+            set => SetValue(MaxValueProperty, value);
+        }
+        #endregion MaxValue Property
+
+
+        #region PressedDateTime Property
+        public static readonly DependencyProperty PressedDateTimeProperty = DependencyProperty.RegisterAttached(
+            "PressedDateTime",
+            typeof(DateTime),
+            typeof(Stepper),
+            new PropertyMetadata(DateTime.MinValue)
+        );
+        public static DateTime GetPressedDateTime(FrameworkElement element)
+            => (DateTime)element.GetValue(PressedDateTimeProperty);
+        public static void SetPressedDateTime(FrameworkElement element, DateTime value)
+            => element.SetValue(PressedDateTimeProperty, value);
+        #endregion PressedDateTime Property
+
+
+
+        #region Value Property
+
+
+        public static readonly DependencyProperty ValueProperty = DependencyProperty.Register(
+            nameof(Value),
+            typeof(double),
+            typeof(Stepper),
+            new PropertyMetadata(0.0, new PropertyChangedCallback((d,e) => ((Stepper)d).OnValueChanged(e)))
+        );
+        protected virtual void OnValueChanged(DependencyPropertyChangedEventArgs e)
+        {
+            Value = Math.Max(MinValue, Value);
+            Value = Math.Min(MaxValue, Value);
+            _valueTextBox.Text = Value.ToString();
+
+            if (Value != (double)e.OldValue && Value == (double)e.NewValue)
+                ValueChanged?.Invoke(this, Value);
+        }
+        public double Value
+        {
+            get => (double)GetValue(ValueProperty);
+            set => SetValue(ValueProperty, value);
+        }
+        #endregion Value Property
+
+
+        public event EventHandler<double> ValueChanged;
+
+        public Stepper()
+        {
+            this.InitializeComponent();
+        }
+
+        private void Border_PointerEntered(object sender, PointerRoutedEventArgs e)
```

Comment:
This class should be removed, but in the future all of this work is done for you by using the Button class